build-system/tasks/make-extension/template/classic/extensions/amp-__component_name_hyphenated__/__component_version__/test/test-amp-__component_name_hyphenated__.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright __current_year__ The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-__component_name_hyphenated__';
+import {createElementWithAttributes} from '../../../../src/dom';
+
+describes.realWin(
+  'amp-__component_name_hyphenated__',
+  {
+    amp: {
+      runtimeOn: true,
+      extensions: ['amp-__component_name_hyphenated__'],
+    },
+  },
+  (env) => {
+    let win;
+    let element;
+
+    beforeEach(() => {
+      win = env.win;
+      element = createElementWithAttributes(win.document, 'amp-__component_name_hyphenated__', {
+        layout: 'responsive',
+      });
+      win.document.body.appendChild(element);
+    });
+
+    it('should contain "hello world" when built', async () => {
+      await element.whenBuilt();
```

Comment:
In this file and others, should we include the `__do_not_submit__` replacement so boilerplate code that doesn't get touched fails CI?

Reply:
I'll add one to the implementation file like on the Bento template. 

Tests though—I think we should make sure they pass after being generated (which I'm doing as a follow-up.)